tests/kola/ignition/delete-config/test.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# kola: { "platforms": "qemu-unpriv" }
+# - platforms: qemu-unpriv
+#   - Ideally we'd test on virtualbox and vmware, but we don't have tests
+#     there, so we mock specifically for ignition.platform.id=qemu
+
+set -xeuo pipefail
+
+. $KOLA_EXT_DATA/commonlib.sh
+
+case "${AUTOPKGTEST_REBOOT_MARK:-}" in
+"")
+    # Ignition boot
+
+    if [ ! -e /run/ignition-rmcfg-ran ]; then
+        fatal "mocked ignition-rmcfg did not run on first boot"
+    fi
+
+    if [ $(systemctl is-active ignition-delete-config ||:) != active ]; then
```

Comment:
Minor:

```suggestion
    if ! systemctl is-active ignition-delete-config; then
```

And below.

Reply:
Yeah, I know, but it seems more readable with the symmetry.